### ISSUE TITLE
SelectField Dropdown Icon not visible in dark mode

### ISSUE THREAD

**Issue by calebpollman** [aws-amplify/amplify-ui#3582]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Other

### How is your app built?

Vite

### What browsers are you seeing the problem on?

Chrome

### Please describe your bug.

<img width="239" alt="image" src="https://user-images.githubusercontent.com/15002885/227070329-b61bbe30-5ea3-4663-a172-1c2e20f97b3c.png">

### What's the expected behaviour?

`SelectField` dropdown icon should be visible in dark mode

### Help us reproduce the bug!

Render a `SelectField` in dark mode

### Code Snippet

````tsx
const theme: Theme = { name: "dark", overrides: [defaultDarkModeOverride] };

const App = () => (
  <ThemeProvider colorMode="dark" theme={theme}>
    <SelectField label="Level of Excitement" />
  </ThemeProvider>
);


### Additional information and screenshots

_No response_

**Comment by reesscot** [comment]
Seems to be showing up in the docs dark mode:
![image](https://user-images.githubusercontent.com/6165315/227589547-05d3eaf7-dc22-4285-8328-83ef6860fbbb.png)
